### ISSUE TITLE
Confusing behavior with _index.md files

### ISSUE THREAD

**Issue by neunenak** [getzola/zola#184]
If I create an `content` directory with a bare `_index.md` file in it, and have a `templates/section.html` template that includes `{{ page.content | safe}}`, I don't see anything at all under the root URL of my site. If I then create a subdirectory of `content` called `subdir` and place a `_index.md` file there, I get an error when I try to build the site

```Error: Failed to render section '/home/greg/code/site/content/sudir/_index.md'
Reason: Failed to render 'section.html'
Reason: Field `page.content` not found in context while rendering 'section.html'
```

In general I'm having a hard time figuring out what the bare minimum of template files is necessary to produce a Gutenberg-generated site with an arbitrary directory structure. I think I could specify an arbitrary template manually per-markdown-file, but I'd like to create my template directory structure in line with the defaults that the content section expects, and I'm having a hard time figuring that out from the documentation.

**Comment by Keats** [comment]
In short:

- `content/_index.md` -> `index.html` (special case)
- all other `_index.md` -> `section.html`
- all pages -> `page.html`

In practice if you don't use taxonomies (tags/categories) those are the only 3 files used. The variables available are explained more in detail in https://www.getgutenberg.io/documentation/templates/pages-sections/
This page is not explaining the index section though, I'll add a paragraph about it.

**Comment by Keats** [comment]
Added a a small bit but there's a PR adding better default templates so that should solve the issue

**Comment by Keats** [comment]
Fixed in 0.3.0 that is currently building in travis/appveyor
